### PR TITLE
Add insights has playbook column

### PR DESCRIPTION
**Pull request by laviro**

### CODE REVIEW

**Comment by laviro** [comment]
@ShimShtein does the idea match your backend work?

File: webpack/InsightsCloudSync/Components/InsightsTable/SelectAllAlert.js [hunk 1]
```diff
@@ -25,7 +24,7 @@ const SelectAllAlert = ({
         )}
         actionLinks={
           <AlertActionLink onClick={selectAll}>
-            {sprintf('Select all %s recommendations', itemCount)}
+            {__('Select recommendations from all pages')}
```
Comment (laviro) [inline webpack/InsightsCloudSync/Components/InsightsTable/SelectAllAlert.js:27]: We decided to drop the actual number of all recommendations, also on other strings here
as we can't really tell if they are selectable.

also, for the `selectAll` functionality which updates the store with the `flag`,
the exact number doesn't really matter.

**Comment by MariaAga** [comment]
Can the table show the disabled rows as selected with `disableSelection` so that the select all at the top of the page will work? this can be needed for example if the user has 24 rows with playbook, 1 without and they click on the select all, all rows will be selected but the user won't have a way to unselect all of them because the select all checkbox is not checked

**Review by MariaAga — APPROVED** [review]
Works with and without playbooks :tada: 
Thanks!

**Comment by ShimShtein** [comment]
Merged. Thanks @laviro!

**Pull request merged**